js/server/modules/@arangodb/foxx/router/validation.js

```diff
@@ -34,20 +33,18 @@ exports.validateParams = function validateParams (typeDefs, rawParams, type) {
     type = 'parameter';
   }
   const params = {};
-  for (const entry of typeDefs) {
-    const name = entry[0];
-    const def = entry[1];
-    if (def.schema && typeof def.schema.validate === 'function') {
```

Comment:
If there's an entry, there's a schema/validate method now, so this check is no longer necessary.